chain/ethereum/src/block_stream.rs

```diff
@@ -833,6 +833,7 @@ where
 
 // This always returns `false` in a normal build. A test may configure reorg by enabling
 // "test_reorg" fail point with the number of the block that should be reorged.
+#[warn(unused_variables)]
```

Comment:
Shouldn't that be `allow`?

Reply:
Of course, this does nothing. Amended the commit.